src/credentials/examples/DeviceAttestationVerifierExample.cpp

```diff
@@ -298,16 +298,14 @@ AttestationVerificationResult ExampleDACVerifier::VerifyAttestationInformation(c
         .vendorId    = 0xFFF1,
         .productId   = 0x8000, // TODO: Retrieve vendorId and ProductId from Basic Information Cluster
         .dacVendorId = dacVendorId,
+        .paiVendorId = dacVendorId,
```

Comment:
This seems wrong to use dacVendorId for both paiVendorId and dacVendorId.

Reply:
I don't see any issues with this assignment. At this point, PAI and DAC VID will be the same.

>VerifyOrReturnError(paiVid == dacVid, AttestationVerificationResult::kDacVendorIdMismatch);
>dacVendorId = static_cast<VendorId>(dacVid);

Reply:
I don't see any issues with this assignment. At this point, PAI and DAC VID will be the same.

> VerifyOrReturnError(paiVid == dacVid, AttestationVerificationResult::kDacVendorIdMismatch);
> dacVendorId = static_cast(dacVid);